packages/flutter/lib/src/material/chip.dart

```diff
@@ -99,6 +101,13 @@ abstract class ChipAttributes {
   /// By default, this is 4 logical pixels at the beginning and the end of the
   /// label, and zero on top and bottom.
   EdgeInsetsGeometry get labelPadding;
+
+  /// Constraints used to increase the gesture detector size without increasing
+  /// the visible material of the chip.
```

Comment:
Maybe note that this increases layout size?

Reply:
Done